src/layouts/Layouts.jsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
+import { useSession } from '@inrupt/solid-ui-react';
 import { NavBar } from '../components/NavBar';
+import { InactivityMessage } from '../components/Notification';
```

Comment:
`import Footer from '../components/Footer/Footer';` also

Reply:
I think we can do this in a future PR. We would want to have Footer in Layouts, but we should consider a few cases when children is too small (we could just force the Footer down with position fixed, for example).

![Screen Shot 2023-05-13 at 12 58 59 (2)](https://github.com/codeforpdx/PASS/assets/14917816/5c402f0f-969c-443a-a0f5-befebe380165)

Or when children is too big (like before, we could just force the Footer up with position fix).

https://github.com/codeforpdx/PASS/assets/14917816/6dbc5c24-8cd9-44e8-b9b7-da1cdbecac90

I was wondering if there's a good way to deal with this in MUI.

Reply:
Hmmmm... I'll see what I can find.  Usually I just toggle a class that gives `position: fixed, bottom: 0` when children are less than viewport height.    And leave it as default `static` when the children are viewport height or longer.   Unlike the Navbar, I don't believe the footer needs to always be on the screen. (Navabr would be sticky.)

Reply:
@leekahung test this ⬆️

This is how I envisioned it. Unlike the `Navbar` (which should probably be `position: sticky:`), the `Footer` I think should be at the bottom of screen when children are shorter than viewport height, but at the bottom of the page content when children are longer than viewport height (the default `position: static`.)

Reply:
<img width="1440" alt="Screenshot 2023-05-14 at 11 27 18 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/ee53d348-3f85-44ed-b533-3933e7b8e816">
<img width="1440" alt="Screenshot 2023-05-14 at 11 27 28 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/7103d3b9-d48b-4e9c-ba78-dfd18ad62c03">

https://github.com/codeforpdx/PASS/assets/71395931/71a1acac-a536-4ee5-82a7-6ef498250247

Reply:
Oh this is great! I thought it'll be more complicated than that. Yeah, I think we can update Layouts based on your suggestion.

Reply:
Updated Layouts and Login as suggested.